app/controllers/idv/session_errors_controller.rb

```diff
@@ -7,16 +7,24 @@ class SessionErrorsController < ApplicationController
     before_action :confirm_idv_session_step_needed
 
     def warning
-      @remaining_step_attempts = remaining_step_attempts
+      @remaining_step_attempts = throttle.remaining_count
+    end
+
+    def failure
+      @expires_at = throttle.expires_at
+    end
+
+    def throttled
+      @expires_at = throttle.expires_at
```

Comment:
`throttled` is for the `:idv_acuant` throttle type, so this isn't quite correct. Will fix in revisions accounting for separate SSN throttle via #5346. I'll also double-check to see why feature tests didn't flag this.

Reply:
> I'll also double-check to see why feature tests didn't flag this.

There were feature specs for this, but they weren't specifically checking the content as showing an expected time remaining. I've added an extra set of assertions in the rebased 1612180 ([see `document_capture_step_spec.rb`](https://github.com/18F/identity-idp/pull/5301/commits/1612180d84337a1a5995182f328dc4f597f477ac#diff-07a84ad3b4f5779028c66ce93ded1dd1ddb0644ae540571c112c242bf9ac95d4)).